File: Bubble Sort/bubblesort.kt
fun bubblesort(arr: IntArray) {
    val n = arr.size
    bool swapped = false
    for (i in 0 until n - 1) {
        for (j in 0 until n - i - 1) {
            if (arr[j] > arr[j + 1]) {
                val temp = arr[j]
                arr[j] = arr[j + 1]
                arr[j + 1] = temp
                swapped = true
            }
        }
        if (swapped == false) {
            break
        }
    }
}

File: Bubble Sort/bubblesort.go
func bubblesort(array[] int)[]int {
	bool swapped = false;
	for i := 0; i < len(array) - 1; i++ {
		for j := 0; j < len(array) - 1 - i; j++ {
			if (array[j] > array[j + 1]) {
				array[j], array[j + 1] = array[j + 1], array[j]
				swapped = true
			}
		}
		if swapped == false {
			break
		}
	}
}